packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -558,24 +570,30 @@ describe('RPC Server Acceptance Tests', function () {
 
                 it('should fail "eth_sendRawTransaction" for London transactions (with insufficient balance)', async function () {
                     const balanceInWeiBars = await servicesNode.getAccountBalanceInWeiBars(accounts[2].accountId);
+                    const gasPrice = await relay.gasPrice();
 
                     const transaction = {
                         ...defaultLondonTransactionData,
                         value: balanceInWeiBars,
                         to: mirrorContract.evm_address,
-                        nonce: await relay.getAccountNonce(accounts[2].address)
+                        nonce: await relay.getAccountNonce(accounts[2].address),
+                        maxPriorityFeePerGas: gasPrice,
+                        maxFeePerGas: gasPrice,
                     };
                     const signedTx = await accounts[2].wallet.signTransaction(transaction);
                     await relay.callFailing('eth_sendRawTransaction', [signedTx], predefined.INSUFFICIENT_ACCOUNT_BALANCE);
                 });
 
                 it('should execute "eth_sendRawTransaction" for London transactions', async function () {
                     const receiverInitialBalance = await relay.getBalance(mirrorContract.evm_address);
+                    const gasPrice = await relay.gasPrice();
 
                     const transaction = {
                         ...defaultLondonTransactionData,
                         to: mirrorContract.evm_address,
-                        nonce: await relay.getAccountNonce(accounts[2].address)
+                        nonce: await relay.getAccountNonce(accounts[2].address),
+                        maxPriorityFeePerGas: gasPrice,
```

Comment:
Similar PR comments for this section as to why it works in local node but is an issue here